nbs/introduction.ipynb

```diff
@@ -85,152 +85,153 @@
    "execution_count": null,
```

Comment:
<p>Why is this here? should we not install NLP in this case?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>